google/resource_compute_instance_test.go

```diff
@@ -1744,6 +1855,16 @@ func testAccCheckComputeInstanceHasMinCpuPlatform(instance *compute.Instance, mi
 	}
 }
 
+func testAccCheckComputeInstanceHasMachineType(instance *compute.Instance, machineType string) resource.TestCheckFunc {
```

Comment:
There was no check about the `machine_type` so I've implemented one (used in `TestAccComputeInstance_statusTerminatedUpdateFields` acceptance tests above). However, I am not fully satisfied with this solution because `instance.MachineType` returns something like, IIRC, `https://googleapis.com/compute/.../projects/<my-project>/zones/<my-zone>/machineTypes/n1-standard-1` and since I cannot access neither project nor zone from the `instance`, I've decided to check only the last part of the machine type (`machineTypes/n1-standard-1`). Is this a suitable solution?